src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
@jquense testing locally, I'm seeing that `updateValueIfChanged` is being called with both a Fiber as well as a DOM node. For example, for our fixture that tests `onChange` events with radio inputs it's called with:

1. The Fiber for the radio input that's being selected
2. The DOM node for the radio input that's being unselected
3. The DOM node for the radio input that's being selected